src/main/java/org/openstreetmap/atlas/tags/MinHeightTag.java

```diff
@@ -0,0 +1,17 @@
+package org.openstreetmap.atlas.tags;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.Tag.Validation;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+
+/**
+ * OSM min_height tag: https://taginfo.openstreetmap.org/keys/min_height#values
+ *
+ * @author ajayaswal
+ */
+@Tag(value = Validation.LENGTH, taginfo = "https://taginfo.openstreetmap.org/keys/min_height#values", osm = "https://wiki.openstreetmap.org/wiki/Key:min_height")
```

Comment:
Is `LENGTH` validator the correct one to use here? TagInfo shows a lot of negative values for this tag (-3, -6, etc.) If we want to consider these, maybe `DOUBLE` is the correct one to use.

Reply:
Also, we may want to add a few unit test cases for this to reflect OSM data (negative values, 0, positives values, decimal values to make sure they're handled properly)

Reply:
Oh also it might be a problem using `Distance` which cannot be negative...

Reply:
Good point, will use a DoubleValidator and add more test cases. Will replace Distance with a new Height scalar.